02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd

```diff
@@ -0,0 +1,413 @@
+---
+title: "Ortholog Mapping - Microarray"
+author: "CCDL for ALSF"
+date: "`r format(Sys.time(), '%B %Y')`"
+output:
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+    df_print: paged
```

Comment:
This will mostly be unnecessary because we render by the script. But if we want it to also show up for the users ini their locally rendering, then I suppose we should keep it, but then we need to add it to all the other modules.

Reply:
If we do want to add this to all headers (which I'm 50-50 on) then we should file a separate "search and replace in the whole project" type issue where we implement this everywhere.e

Reply:
As noted by Josh in a Slack discussion, the notebook headers should not need to be changed as `paged_df` is the default for `html_notebook` so it will look the same either way for people running locally. 

I'll remove it from the notebook header in this PR but keep the change to the `render-notebooks.R` script so that the html output files render with this option.